src/main/templates/contentmodifiers.hbs

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  {{>header}}
+  <!-- Main content starts - do not edit above this line -->
+    <table id="sorttable" class="table table-small table-hover display">
+      <thead class="thead-light">
+        <tr>
+          <th scope="col" class="align-text-top d-none">Sort Order</th>
+          <th scope="col" class="align-text-top" width="10%">DCNC Code <small>(For CTT)</small></th>
+          <th scope="col" class="align-text-top" width="20%">Description</th>
+          <th scope="col" class="align-text-top" width="10%">CPL Metadata Type</th>
+          <th scope="col" class="align-text-top text-wrap">CPL Metadata Sample</th>
+          <th scope="col" class="align-text-top text-wrap">Metadata Scope</th>
+          <th scope="col" class="align-text-top">Metadata Defining Document</th>
+          
+        </tr>
+      </thead>
+      <tbody id="searchTable">
+        {{#data}}
+        <tr id="{{../idType}}-{{dcncCode}}">
+          <td class="d-none">
+            {{dcncSortOrder}}
+          </td>
+          <td class="code"><a class="anchor" id="{{dcncCode}}" href="#{{dcncCode}}"></a><code>{{dcncCode}}</code></td>
```

Comment:
Suggest using the same color/font style as the value in the CPL column

Reply:
Yea, I was debating this. I think it's important for the DCNC codes to be in a different color so that it's obvious to the reader that the red'ish code values are DCNC, whereas the dark grey are metadata. I'm also partially apt to making all values that need to be entered the red'ish, and keep descriptions in black. Thoughts?

Reply:
> reader that the red'ish code values are DCNC, whereas the dark grey are metadata

well... but some of the values are the same in CTT and CPL, in which case it would be ok for them to be the same color, right?

Reply:
> > reader that the red'ish code values are DCNC, whereas the dark grey are metadata
> 
> well... but some of the values are the same in CTT and CPL, in which case it would be ok for them to be the same color, right?

Would it make more sense to have clear separation of values that go in CTT and CPL metadata, or clear separation of user enterable info (for both CTT and CPL) and descriptive info in the table?

Reply:
@palemieux see update at 1e9af3f for all user entry values updated to show as `<code>` class as what I mean in prior comment.

Reply:
I would think that each row would be about one metadata value, e.g. luminance. The row would contain:

* a description of the metadata value
* a mapping of the metadata value to CTT
* a CTT example
* a mapping of the metadata value to CPL metadata
* a CPL metadata example

Reply:
These have been updated with 770236f